### DISCUSSION TITLE
How to bypass anti DLL

### DISCUSSION THREAD

**Discussion by hungdz1234** [danielkrupinski/Osiris#2757] [Q&A]
Hi, I inject Osiris into CS:GO and get banned by this server: 103.9.158.241:27019
![image](https://user-images.githubusercontent.com/54879097/113382247-3f8fa000-93ab-11eb-8eec-481c22343bf0.png)

Looks like they're using some kind of antiDLL method to ban whoever using DLL. Is there anything I can do to bypass this? If I change my injector or injection method, can I bypass this?

**Comment by kdb17wsl** [comment]
you'd better play the official server.

**Comment by maecry** [comment]
that just flagged the event listener lol just remove event listener hook and bet u will be fine

**Comment by gicung91** [comment]
how? i need play face it
